src/views/documents/utils/PrintButton.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <a target="_blank" :href="path">
+    <button class="button is-primary is-size-7-mobile">Print search results</button>
```

Comment:
![image](https://user-images.githubusercontent.com/2234024/201624410-a7a63f8a-2964-4cde-b6ae-d8ea44852383.png)

Maybe we could replace the text with a print icon on mobile, to save some space?